src/lib/support/DefaultStorageKeyAllocator.h

```diff
@@ -45,7 +45,11 @@ class DefaultStorageKeyAllocator
     // FailSafeContext
     const char * FailSafeContextKey() { return Format("g/fsc"); }
 
-    // Access Control List
+    // Access Control
+    const char * AccessControlExtensionEntry(size_t version, FabricIndex fabric)
+    {
+        return Format("a/%x/1/%x", static_cast<unsigned>(version), static_cast<unsigned>(fabric));
```

Comment:
This clashes with other established format:
```
const char * AttributeValue(const app::ConcreteAttributePath & aPath)
    {
        // Needs at most 24 chars: 4 for "a///", 4 for the endpoint id, 8 each
        // for the cluster and attribute ids.
        return Format("a/%" PRIx16 "/%" PRIx32 "/%" PRIx32, aPath.mEndpointId, aPath.mClusterId, aPath.mAttributeId);
    }
```

Since this is 1 entry per fabric (max) and fabric-scoped, suggest following the format of other fabric-scoped attributes:

`f/%x/acle/%x` with FabricIndex as first arg and your version if you want it as second arg. That keeps the same tree hierarchy

Reply:
I thought 'a' was unused I missed it. I'll have to change the key, I will revisit the format.

Reply:
Now it's 'f/<fabricindex>/ac/1'.